charts/seed-bootstrap/dashboards/alerts-dashboard.json

```
@@ -429,7 +429,7 @@
       "pluginVersion": "6.3.2",
       "targets": [
         {
-          "expr": "(count(count(ALERTS) by (cluster)) / count(count(seed:container_cpu_usage_seconds_total:sum_by_pod{namespace=~\"shoot-(.+)\"}) by (namespace))) * 100",
+          "expr": "count(count(ALERTS)by(cluster)) / sum(seed:active_shoots_total:sum) * 100",
```

nit: keep the whitespace around the `by` keyword for readability

```suggestion
          "expr": "count(count(ALERTS) by (cluster)) / sum(seed:active_shoots_total:sum) * 100",
```

---

Done! :)